Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/storage/entity/ContactDiaryCoronaTestEntity.kt

```diff
@@ -38,7 +38,7 @@ data class ContactDiaryCoronaTestEntity(
 }
 
 fun CoronaTest.canBeAddedToJournal(): Boolean {
-    return isViewed && (isNegative || isPositive)
+    return isNegative || (isViewed && isPositive)
```

Comment:
Can you please add  a comment here why did you change it ?

Reply:
sure